src/components/CustomButton.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from "@emotion/styled";
+
+const StyledButton = styled.button`
+  background-color: ${(props) => props.buttonColor};
+  border: none;
+  color: ${(props) => props.theme.colors.white};
+  padding: ${(props) => props.theme.layout.space200}
+    ${(props) => props.theme.layout.space500};
+  text-align: center;
+  text-decoration: none;
+  font-size: 16px;
+  cursor: pointer;
+  border-radius: 5px;
+
+  &:hover {
+    background-color: ${(props) => props.hoverColor};
+  }
+`;
+
+const CustomButton = ({ buttonName, buttonColor, hoverColor, buttonText }) => {
```

Comment:
I appreciate the flexibility of this implementation with color props. However, to improve maintainability and align with our project's component-based architecture, I suggest creating two distinct components instead of passing in colors as props:

- `PrimaryButton` (for Button 1 style)
- `SecondaryButton` (for Button 2 style)

I've attached a screenshot below for reference. Let me know if you have any additional thoughts on this approach!

![Screenshot 2024-07-22 at 7 15 39 PM](https://github.com/user-attachments/assets/d0408c1d-a69e-48c8-87ce-e8a328334e2e)